Renaming follow-up changes

Wapwolf opened the pull request:
A very important and life-changing submission

## Before submitting (must do checklist)

- [x] Did you read the [contribution guide](https://github.com/etna-team/etna/blob/master/CONTRIBUTING.md)?
- [ ] Did you update the docs? We use Numpy format for all the methods and classes.
- [ ] Did you write any new necessary tests?
- [ ] Did you update the [CHANGELOG](https://github.com/etna-team/etna/blob/master/CHANGELOG.md)?
<!-- For CHANGELOG separate each item in unreleased section by blank line to reduce collisions -->

## Proposed Changes
<!-- Add a more detailed description of the changes if needed. 
No need for typos and docs improvements  -->
* Fixed the naming and link to the company after renaming from tinkoff to t-bank

## Closing issues
<!-- Link Issue you are closing here. 
Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such). -->

d-a-bunin commented:
Hi! What do you think about updating the license? 

https://github.com/etna-team/etna/blob/master/LICENSE#L178

Wapwolf commented:
tbh me too, this is a question for the IP lawyer. But my guess would be that yes, it should also be updated

d-a-bunin commented:
I have in mind some other places to update:
- `pyproject.toml`
  - Description field
  - Emails for the initial developers
- `docs/source/conf.py`
  - `copyright`
  - `author`
- `docs/source/index.rst`

Wapwolf commented:
Fixed

> I also found some other places to update:
> 
> * `pyproject.toml`
>   
>   * Description field
>   * Emails for the initial developers
> * `docs/source/conf.py`
>   
>   * `copyright`
>   * `author`
> * `docs/source/index.rst`

Fixed

d-a-bunin commented on LICENSE:
```diff
@@ -175,7 +175,7 @@
    END OF TERMS AND CONDITIONS
 
-   Copyright 2021 Tinkoff Bank
+   Copyright 2021 T-Bank
```
Wasn't it decided not to do this renaming in LICENSE?

Wapwolf replied:
Hmm, forgot to pull, sorry. Fixed in newest commit

d-a-bunin replied:
It seems like it wasn't fixed by the newest commit. The most reliable source of a change is a tab "Files changed" in the PR.

Wapwolf replied:
Well, now :Files changed" tab does not even show the change in LICENSE. But I guess I fixed my mistake, now it surely has the correct version

d-a-bunin commented on docs/source/index.rst:
```diff
@@ -8,7 +8,7 @@ to SOTA neural networks, models combination methods and smart backtesting.
 
 ETNA is the first python open source framework of
-`Tinkoff.ru <https://www.tinkoff.ru/eng/>`_ Artificial Intelligence Center.
+`Tinkoff.ru <https://www.tbank.ru/>`_ Artificial Intelligence Center.
```
Probably, Tinkoff.ru here should also be renamed.

Wapwolf replied:
Fixed
